modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -356,12 +356,18 @@ def confinements_dont_overlap?(confinements)
         end
 
         def validate_form_526_veteran_homelessness!
-          if too_many_homelessness_attributes_provided? || unnecessary_homelessness_point_of_contact_provided?
-            raise ::Common::Exceptions::InvalidFieldValue.new(
-              'homelessness',
-              form_attributes['veteran']['homelessness']
+          if too_many_homelessness_attributes_provided?
+            raise ::Common::Exceptions::UnprocessableEntity.new(
+              detail: "Must define only one of 'veteran.homelessness.currentlyHomeless' or 'veteran.homelessness.homelessnessRisk'" # rubocop:disable Layout/LineLength
```

Comment:
Could you put this on multiple lines to improve readability and to avoid disabling the cop?

```suggestion
              detail: "Must define only one of 'veteran.homelessness.currentlyHomeless' or "\
                      "'veteran.homelessness.homelessnessRisk'"
```